frontend/src/pages/RecipientRecord/pages/MergeGoals/components/GoalMergeGuidanceDrawer.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Drawer from '../../../../../components/Drawer';
+
+export default function GoalMergeGuidanceDrawer({ drawerTriggerRef }) {
+  if (!drawerTriggerRef) {
+    return null;
+  }
+
+  return (
+    <Drawer
+      triggerRef={drawerTriggerRef}
+      stickyHeader
+      stickyFooter
+      title="Merging goals"
+    >
+      <p className="usa-prose margin-top-0">
```

Comment:
Do we want to pull this guidance from jira?

Reply:
Probably, but I asked and it's not in there yet